src/components/users/user.routes.ts

```diff
@@ -1,6 +1,8 @@
 import { Router } from 'express'
+import { check } from 'express-validator'
 
-import { isAuthenticated, isAdmin } from '../../config/passport'
+import { isAdmin, isAuthenticated } from '../../config/passport'
+import { handeValidationError } from '../../util/errorHandlers'
```

Comment:
Oh, no! This `handeValidationError` seems to be an old typo back from April. I guess, right now, we should dismiss this problem, because #323 uses this function as well.

Reply:
Shall I push an issue just to be sure not to forget about this? We can solve it after merging these two pull requests.

Reply:
Nice catch! Yes, please create an issue for this.